index.js

```diff
@@ -29,6 +29,17 @@ class Glossary {
     this._entries[term] = description
   }
 
+  fromFile (filename) {
+    let rawdata = fs.readFileSync(filename)
+    let glossary = JSON.parse(rawdata)
+    for (var i in glossary) {
+      assert(i && i.length, `term is required for description ${glossary[i]}`)
```

Comment:
Rather than re-doing this validation, we should just use the existing `.add()` method and get the existing validation for free.